src/config/cartFunctions.js

```diff
@@ -0,0 +1,111 @@
+// ------------------------------------------
```

Comment:
Is there an atom in featuresConfig to turn on or off the cart? Not everyone will want to show it

Reply:
I think like we said with @HugoWith it can be always active it still work because it's not linked to an attribute or an index,...

Reply:
My Bad Alex, I had a poor judgement on this, I thought it won't be a problem for SE to have it active. Let's create an Atom then, I can do it, no worries 
@VirAlex I think we should delete the PR as the branch is not fully ready to be merged. I'd like to work on some issues as well. What do you think ?

Reply:
I think we should create an atom yes but keep open this PR to have the history of it, actually all @matthew1809 have been fixed because since this comment we continued to work on it what's missing actually is the lazy investigation @HugoWith

Reply:
✅

Reply:
Thanks a lot @VirAlex 🙏

Reply:
https://github.com/algolia/algolia-react-boilerplate/pull/436/commits/db5d9dac04855b9ff13a239caa64cf8934a3b2b1
Done ✅ @matthew1809 @HugoWith